guides/manage_data/Introduction_to_Artifacts.ipynb

```diff
@@ -24,13 +24,11 @@
     "\n",
     "We’ve built Comet Artifacts to help Machine Learning teams solve the challenges of iterating on datasets and tracking pipelines where the data generated from one experiment is fed into another experiment.\n",
     "\n",
-    "An Artifact is a composed of Artifact Versions. Each Artifact has a name, a type, description, tags, and metadata\n",
+    "An Artifact is composed of Artifact Versions. Each Artifact has a name, a type, description, tags, and metadata.\n",
     "\n",
     "An Artifact Version is a snapshot of files & assets, arranged in a folder-like logical structure. This snapshot can be tracked using metadata, a version number, tags and aliases. A Version tracks which experiments consumed it, and which experiment produced it.\n",
     "\n",
-    "For a more complete overview [check out our full annoucement here](https://www.comet.ml/site/announcing-comet-artifacts/?utm_campaign-artifacts-launch&utm_source=colab-example&utm_medium=additional-resources)\n",
-    "\n",
-    "\n"
+    "For a more complete overview [check out our full annoucement here](https://www.comet.ml/site/announcing-comet-artifacts/?utm_campaign-artifacts-launch&utm_source=colab-example&utm_medium=additional-resources)\n"
```

Comment:
```suggestion
    "For a more complete overview [check out our full announcement here](https://www.comet.ml/site/announcing-comet-artifacts/?utm_campaign-artifacts-launch&utm_source=colab-example&utm_medium=additional-resources)\n"
```

Reply:
This link is dead, as you noted. I don't have a better one right now. Do we need it at all?